src/languages/es.js

```diff
@@ -150,7 +151,9 @@ export default {
         deleteConfirmation: '¿Estás seguro de que quieres eliminar este comentario?',
     },
     reportActionsView: {
-        beFirstPersonToComment: 'Sé el primero en comentar',
+        beginingOfChatHistroy: 'This is the beginning of your chat history with',
+        beginingOfChatHistroyPrivate: 'This is the beginning of the private',
+        beginingOfChatHistroyPrivateSectionPart: 'room, invite others by @mentioning them.',
```

Comment:
Please use proper translations and you can ask them in the Slack channel or on the issue.

Reply:
@parasharrajat proper translation on es.js side?
or in English side

Reply:
ok asked on issue

Reply:
as i get answer i will update